src/profile/helpers/getAddress.ts

```diff
@@ -1,9 +1,20 @@
+import countries from 'i18n-iso-countries';
+import fi from 'i18n-iso-countries/langs/fi.json';
+import en from 'i18n-iso-countries/langs/en.json';
+import sv from 'i18n-iso-countries/langs/sv.json';
+
 import { MyProfileQuery } from '../../graphql/generatedTypes';
 
-export default function getAddress(data: MyProfileQuery) {
+countries.registerLocale(fi);
```

Comment:
You are configuring this module in the same way in two places 🤔Maybe there's a petter which would allow you to only do it once?

Reply:
Good point. I moved registerLocale stuff to App.tsx, seems to be working.